change/@ni-nimble-angular-040b8f50-1a12-4746-b73d-85a5a4b97f3a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Note: We won't want to merge this change until we've done the necessary app work to be ready to update the whole platform to Angular 15.